x11/windowmaker/Portfile

```
@@ -38,26 +48,20 @@ depends_lib \
 	port:giflib \
 	port:tiff
 
-post-patch {
-	set fl [glob -directory ${worksrcpath} WindowMaker/*menu*]
-	lappend fl ${worksrcpath}/WindowMaker/Defaults/WindowMaker.in \
-	               ${worksrcpath}/WPrefs.app/Menu.c ${worksrcpath}/WPrefs.app/Paths.c
-	foreach i $fl {
-		reinplace "s|/usr/X11R6/share|${prefix}/share|g" $i
-		reinplace "s|/usr/share|${prefix}/share|g" $i
-		reinplace "s|/usr/local/share|${prefix}/share|g" $i
-		reinplace "s|/opt/share|${prefix}/share|g" $i
-		reinplace "s|/opt/kde2/share|${prefix}/share|g" $i
-	}
-}
+configure.env   LINGUAS="bg cs da de el es et fi fr gl hr hu it ja \
+                ko ms nl no pl pt ro ru sk sv tr zh_CN"
 
-configure.env   "LINGUAS=bg cs da de el es et fi fr gl hr hu it ja \
-                ko ms nl no pl pt ro ru sk sv tr zh_CN" \
-                "LIBS=-L${prefix}/lib -lintl -lX11"
 configure.args  --mandir=${prefix}/share/man \
-                --with-nlsdir=${prefix}/share/locale \
-                --with-gnustepdir=${prefix}/GNUstep/Local
+    --with-gnustepdir=${prefix}/GNUstep/Local \
+    --disable-magick 
 
+variant debug description "Build with debugging flags set" {
+    # To keep .c/.o/.dSYM files around, set portautoclean no in macports.conf
+    # or use port -k to “keep” files around.
+    configure.args-append --enable-debug
+    configure.env-append CFLAGS=-g2 -O0
```

Don't manually put `CFLAGS` into `configure.env`. MacPorts already does it for you, putting the contents of `configure.cflags` into `CFLAGS`. If you want to change what it puts there, change `configure.cflags`.

`-O0` is an optimization flag. MacPorts stores its default optimization flag in `configure.optflags`, and that becomes a part of `configure.cflags`. If you want different optimization flags, override `configure.optflags`.

```suggestion
    configure.optflags -O0
    configure.cflags-append -g2
```